checador/app/views/devise/sessions/new.html.erb

```diff
@@ -1,27 +1,22 @@
-<%= render 'layouts/flash' %>
-<h2>Log in</h2>
+<div class="container w-25">
+  <%= render 'layouts/flash' %>
+  <h2 class="text-center">Log in</h2>
 
-<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
-  <div class="field">
-    <%= f.label :email %><br />
-    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
-  </div>
-
-  <div class="field">
-    <%= f.label :password %><br />
-    <%= f.password_field :password, autocomplete: "current-password" %>
-  </div>
+  <%= form_for(resource, as: resource_name, url: session_path(resource_name)) do
+  |f| %>
```

Comment:
Not important, but just to avoid confusion, I think this code could be better in one line, what do you think?

Reply:
Yes, you're right 👍